python-threatexchange/threatexchange/cli/cli_config.py

```diff
@@ -214,33 +258,13 @@ def get_signal_types_for_content(
     ) -> t.List[t.Type[signal_base.SignalType]]:
         return self._mapping.signal_and_content.signal_type_by_content[content_type]
 
-    def get_fetchers(self):
-        return [fs for fs in self._mapping.fetcher.fetchers_by_name.values()]
-
-    def get_api_for_collab(
-        self, collab: collab_config.CollaborationConfigBase
-    ) -> SignalExchangeAPI:
-        return self._mapping.fetcher.fetchers_by_name[collab.api]
-
-    def get_fetch_store_for_fetcher(
-        self, fetcher: t.Type[SignalExchangeAPI]
-    ) -> FetchedStateStoreBase:
-        return CliSimpleState(fetcher, self._state.dir_for_fetched_state(fetcher))
-
-    def get_fetch_store_for_collab(
-        self, collab: collab_config.CollaborationConfigBase
-    ) -> FetchedStateStoreBase:
-        return self.get_fetch_store_for_fetcher(
-            self._mapping.fetcher.fetchers_by_name[collab.api].__class__
-        )
-
     @property
     def in_demo_mode(self) -> bool:
         """Has no live collabs"""
         return not self._state.get_all_collabs()
 
     def get_all_collabs(
-        self, *, default_to_sample: bool = False
+        self, *, default_to_sample: bool = True
```

Comment:
(nonblocking) : I found tracing through the calls of where `default_to_sample` is used a tad confusing. 
I think it would be clearer if you either 1) add a docstring for the methods? or 2) make the argument even more verbose something like`default_to_using_only_sample_if_demo` 

unsure: one thing that might make things clearer is if you switch the method to default to false. e.g. the expected usage of this method is production not demo and where the demo state calls occur use the flag.

Reply:
Not sure more verbose is more clear. When I changed the flag I considered renaming the variable. I plan on continuing a previous PR's switching from settings.get_y() => settings.y.get() for better grouping, and can make a stab at renaming then. 

Re: setting back to default to false - The more common expected behavior is that you want to use the sample setting, there's probably only one case where you don't (list which configs you've configured). Nasty hard-to-detect bugs will be created if you forget to set the flag to true, so it seems better to default to the more safe/expected behavior.